github_activity/github_activity.py

```diff
@@ -55,24 +55,30 @@ def get_activity(target, since, until=None, repo=None, kind=None, auth=None):
     until_dt, until_is_git_ref = _get_datetime_and_type(org, repo, until)
     since_dt_str = f"{since_dt:%Y-%m-%dT%H:%M:%SZ}"
     until_dt_str = f"{until_dt:%Y-%m-%dT%H:%M:%SZ}"
-    search_query += f' updated:{since_dt_str}..{until_dt_str}'
 
     if kind:
         allowed_kinds = ['issue', 'pr']
         if kind not in allowed_kinds:
             raise ValueError(f"Kind must be one of {allowed_kinds}")
         search_query += f" type:{kind}"
 
+    # Query for both opened and closed issues/PRs in this window
     print(f'Running search query:\n{search_query}\n\n')
-    qu = GitHubGraphQlQuery(search_query, auth=auth)
-    qu.request()
-    qu.data.since_dt = since_dt
-    qu.data.until_dt = until_dt
-    qu.data.since_dt_str = since_dt_str
-    qu.data.until_dt_str = until_dt_str
-    qu.data.since_is_git_ref = since_is_git_ref
-    qu.data.until_is_git_ref = until_is_git_ref
-    return qu.data
+    query_data = []
+    for activity_type in ['created', 'closed']:
+        ii_search_query = search_query + f' {activity_type}:{since_dt_str}..{until_dt_str}'
```

Comment:
this ensures that if an old PR is "updated" outside the window we search, it will still be found. In its current form, if 20 PRs were updated in January, and then somebody added a tag to each of them in March, then doing a search for the month of January would have not found any of those PRs